test/common/test_common.cpp

```diff
@@ -253,172 +274,297 @@ TEST (PCL, PointCloud)
     }
   }
 #endif
+}
+
+TEST_F (pointCloudTest, pointCloud_clear)
```

Comment:
No need to repeat `pointCloud` in the `pointCloud_clear` style names in all tests